src/json-editor.component.html

```diff
@@ -9,7 +9,7 @@
   </div>
   <div id="editor-middle-column" [ngClass]="{widerColumn6: isPreviewerHidden}" class="col-md-6 middle" [shortcuts]="shortcuts">
     <table id="editor" class="table">
-      <div class="field-wrapper" *ngFor="let key of keys | addAlwaysShowFields:schema | filterAndSortBySchema:schema; trackBy:trackByElement">
+      <div class="field-wrapper" *ngFor="let key of keys | addAlwaysShowFields:schema | filterAndSortBySchema:schema; let last=last; trackBy:trackByElement" [ngClass]="{last:last}">
```

Comment:
With the css selectors I can get all the last element but not the very last... Because the last-of-type selector selects every last element... Other than that the problem isn't to apply some css but recognize it in order to check every time on tooltip shown if the placement of the tooltip must be top or bottom...